packages/reports/src/Helpers/market-underlying.ts

```diff
@@ -26,11 +36,15 @@ export const getMarketInformation = shortcode => {
     return market_info;
 };
 
-export const getMarketName = underlying => (underlying ? getMarketNamesMap()[underlying.toUpperCase()] : null);
+export const getMarketName = (underlying: string | null) =>
+    underlying ? getMarketNamesMap()[underlying.toUpperCase() as keyof typeof getMarketNamesMap] : null;
```

Comment:
```suggestion
export const getMarketName = (underlying: keyof ReturnType<typeof getMarketNamesMap>) =>
    getMarketNamesMap()[underlying];
```
@akmal-deriv Same for `getTradeTypeName` method, I'm not sure, Would you please check them 🙏🏻

Reply:
I'm following the implementation for the same functions in shared package, so they provide `string` type to the inputs. I don't think we should have different types for the same functions across different packages🙏🏻